src/c76_menuCAL.hpp

```diff
@@ -168,6 +168,17 @@ bool checkProgressiveUpDown(int *val, int minDelay = 25)
   return ret;
 }
 
+bool checkProgressiveUpDown(int *val, int minDelay = 25)
+{
+  long lval = *val;
+  bool ret = checkProgressiveUpDown(&lval, minDelay);
+  if ((lval < 32766L) && (lval > -32765L))
```

Comment:
please use `INT16_MIN` and `INT16_MAX` instead of magic numbers

Reply:
I could not get any constants to compile across all platforms, which is why I ended up using these...

Reply:
Fixed